### DISCUSSION TITLE
how to turn off auto syntax completion?

### DISCUSSION THREAD

**Discussion by jjgitit** [NvChad/NvChad#2687] [Q&A] [answered]
Hi all, 
I having a trouble looking at my own code a lot of the times because whenever I write some object, Nvchad automatically toggles parameter toggle(i've attached the photo because i don't think that's the correct name). If anyone could help me configure this, it would greatly appreciated! Thanks.
![Screenshot 2024-02-24 at 01 15 08](https://github.com/NvChad/NvChad/assets/132883866/2c4edac9-bbe9-4662-81ab-a3c24e2b15b8)

**Comment by siduck** [comment, marked as answer]
turn off signature option, in your chadrc.   pls read core/default_config.lua
